manifests/v/VirtualHere/USBClient/5.8.2/VirtualHere.USBClient.installer.yaml

```
@@ -10,9 +10,6 @@ Commands:
 Installers:
 - Architecture: arm64
   InstallerUrl: https://www.virtualhere.com/sites/default/files/usbclient/vhuiarm64.exe
-  InstallerSha256: 88560789CB5698FE61B7B09A69321388D3EA53CF5A7CAE5115E9A43DFF9EBE16
-- Architecture: x64
-  InstallerUrl: https://www.virtualhere.com/sites/default/files/usbclient/vhui64.exe
-  InstallerSha256: 03EEC385DC313922331734EAC1931A77D27EE6B4F68D2FCD44FC829A6E871FE2
+  InstallerSha256: 2754ACE3D2D7ABAB1268D7ED1C8C194BC479233A3734FCDCE2378F7582315418

```

```suggestion
 InstallerSha256: 9E1A7D1D70BAF5E285C87BC839E52A4BC0C55234CCED6E2FE4F21FF5ED9729EA
```

(Automated response - build 978.)